datafusion/core/src/physical_plan/sorts/sort_preserving_merge.rs

```diff
@@ -196,16 +197,23 @@ impl ExecutionPlan for SortPreservingMergeExec {
                             part_i,
                             context.clone(),
                         );
-                        (receiver, join_handle)
+
+                        SortedStream::new(
+                            RecordBatchReceiverStream::create(
+                                &schema,
+                                receiver,
+                                join_handle,
+                            ),
+                            0,
```

Comment:
I think specifying 0 here is ok, but perhaps @yjshen could confirm?

Reply:
Yes. This field is `mem_used` for `SortedStream`. This 0 is consistent with what we report for Receivers. We currently do not count in memory usage for a single batch for all streams' `next()`.